test/metrics-luatest/helper.lua

```diff
@@ -0,0 +1,13 @@
+-- Assert running tests with built-in metrics.
+local rock_utils = require('third_party.metrics.test.rock_utils')
+rock_utils.assert_builtin('metrics')
+
+-- Workaround paths to reuse existing submodule tests.
+local function workaround_requires(path)
+    package.preload[path] = function()
+        return require('third_party.metrics.' .. path)
+    end
+end
+
+workaround_requires('test.utils')
+workaround_requires('test.psutils_linux_test_payload')
```

Comment:
Raw idea: can we walk over `test/metrics-luatest/*_test.lua` and `third_party/metrics/test/**/*_test.lua` and perform some check that all the tests are included into tarantool? We can write the checks as a test, for example. Now it looks easy to miss a new test at the submodule update.

Reply:
Thanks!